.github/workflows/ci.yml

```diff
@@ -27,11 +25,12 @@ jobs:
           server-password: OSSRH_TOKEN # env variable for token in deploy
           gpg-private-key: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }} # Value of the GPG private key to import
           gpg-passphrase: MAVEN_GPG_PASSPHRASE # env variable for GPG private key passphrase
-      - name: Publish alfresco-value-assistance-repo packages
-        if: ${{ startsWith(github.ref, 'refs/heads/master') }}
-        run: mvn --batch-mode deploy
+      #build all artifacts.
+      - name: Build alfresco-value-assistance packages
+        if: ${{ !startsWith(github.ref, 'refs/heads/master') }}
+        run: mvn clean install
         working-directory: alfresco-value-assistance-repo
```

Comment:
Is alfresco-value-assistance-share built too? Should `working-directory` not be removed?